openbb_terminal/stocks/fundamental_analysis/fa_controller.py

```diff
@@ -781,11 +781,32 @@ def call_fraud(self, other_args: List[str]):
             default=False,
             help="Shows an explanation for the metrics",
         )
+        parser.add_argument(
+            "-d",
+            "--detail",
+            action="store_true",
+            dest="detail",
+            default=False,
+            help="Shows the details for calculating the mscore",
+        )
+        parser.add_argument(
```

Comment:
let's have it true by default

Reply:
Are you sure? These calculations are kind of "in the weeds" and will not be valuable to most people.